client/finality-grandpa/src/authorities.rs

```diff
@@ -631,6 +686,36 @@ impl<H, N: Add<Output=N> + Clone> PendingChange<H, N> {
 	}
 }
 
+// Tracks authority set changes. We store the block numbers for the first block of each authority
```

Comment:
We should make it clear here that this tracks historical authority set changes that have already happened. Not to be confused with the rest of the fields above where we are tracking possible upcoming authority set changes.